.ci-builds/.azure-pipelines.yml

```diff
@@ -44,8 +44,8 @@ jobs:
     vmImage: 'Ubuntu 16.04'
   steps:
     - script: |
-        export CPPFLAGS="-Wall -Werror" && cd icu4c/source && ./runConfigureICU Linux && make -j2 check
-      displayName: 'Build and Test (WarningsAsErrors)'
+        export CPPFLAGS="-Wall -Werror -Wextra" && cd icu4c/source && ./runConfigureICU Linux && make -j2
```

Comment:
Please use `make -j2 tests` so that the tests are also built.

Reply:
Alternative: if the tests have too many warnings and you don't want to clean them up, add a comment saying something to that extent and keep it at just building the library code.

Reply:
Thanks Shane, that's a good point. I'll see about fix the warnings/errors in the tests.